docs/sources/panels-visualizations/visualizations/pie-chart/index.md

```diff
@@ -35,6 +35,21 @@ The following video guides you through the creation steps and common customizati
 
 {{< docs/play title="Grafana Bar Charts and Pie Charts" url="https://play.grafana.org/d/ktMs4D6Mk/" >}}
 
+## Supported data formats
+
+The pie chart visualization requires a dataset with at least one row of numeric values to be added together. If the first row contains column titles, it's used to display and identify the values in the pie chart. Cells with non-numeric values are not displayed in the pie chart.
+
+Example:
+
+| Description   | Internet Explorer | Chrome |
+| ------------- | ----------------- | ------ |
+| Other users   | 1.5               | 7      |
+| English users | 17.5              | 50     |
+
+If the dataset contains multiple rows, the visualization displays the last row by default, but multiple rows can be configured as described in the [Value options](#value-options) section. If you configure multiple rows to be displayed, the panel concatenates the text fields of each row with the numeric column titles.
+
+It is recommended to use only one data set with the pie chart, if multiple data sets are present the numeric values configured in the _Value options_ section will be added up and displayed from all the data sets.
```

Comment:
I've edited this a bit, but I'm still not exactly sure what it means. Do you mean the numeric values of all the datasets configured in the Value options section will be added together?

```suggestion
We recommend you only use one data set with the pie chart. If you have multiple datasets, the numeric values configured in the **Value options** section will be added up and displayed from all the datasets.
```